db/seeds.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
-Dir[Rails.root.join("db/seeds/initial_seed.rb")].each { |seed| load seed }
+load Rails.root.join("db/seeds/initial_seed.rb")
```

Comment:
Add `.to_s` to fix it in integration tests. I haven't figured out why, cause it looks like a deep and horrible dive, but that's what I've done on E&L.